akka-docs/rst/scala/code/docs/stream/TwitterStreamQuickstartDocSpec.scala

```diff
@@ -31,11 +31,8 @@ object TwitterStreamQuickstartDocSpec {
   val akka = Hashtag("#akka")
   //#model
 
-  // format: OFF
   //#tweet-source
-  val tweets: Source[Tweet, NotUsed]
-  //#tweet-source
-  // format: ON
+  val tweets: Source[Tweet, NotUsed] //#tweet-source
```

Comment:
(this breaks the docs, and not having the format: OFF there makes scalariform re-break it for every PR)